src/index.js

```diff
@@ -289,11 +291,12 @@ export default function undoable (reducer, rawConfig = {}) {
           }
         }
 
-        if (history.present !== res) {
+        // Don't add entry to history if it was just created
+        if ((justCreatedHistory && history.present === res) || action.type === '@@redux-undo/INIT') {
```

Comment:
> If we use a flag that is set to true after the first run of the reducer, then a hot-reload which calls @@redux/INIT will have an incorrect flag and will call an extra insert.

The flag is in the scope of the function, so that won't cause an extra insert right?

And HMR will always replace the whole reducers and reapply our decorator around the recuder right? 

    module.hot.accept('../reducers', () => {
      const nextRootReducer = require('../reducers').default
      store.replaceReducer(nextRootReducer)
    })